README.md

```diff
@@ -1,492 +1,271 @@
 
-Please feel free to contribute, we do accept pull requests. This solution has been built using VS2017, you will need it to open this project.
+-   ⚡ Lightning fast
+-   🎈 Effortless full-stack integration testing - from the front-end to the back-end
+-   🔌 Supports HTTP/REST and event-driven systems
+-   🛠️ Configurable mock server
+-   😌 Powerful matching rules prevents brittle tests
+-   🤝 Integrates with Pact Broker / Pactflow for powerful CI/CD workflows
+-   🔡 Supports 12+ languages
 
-## History
-PactNet was initially built at [SEEK](https://www.seek.com.au/) to help solve some of the challenges faced with testing across service boundaries.  
-The project now lives in the pact-foundation GH organisation, to help group and support the official Pact libraries.  
-Massive thanks to the SEEK team for all the time and hard work put into this library.
+**Why use Pact?**
 
-[![SEEK](https://raw.githubusercontent.com/pact-foundation/pact-net/master/seek.png "SEEK")](https://www.seek.com.au/)
+Contract testing with Pact lets you:
 
+-   ⚡ Test locally
+-   🚀 Deploy faster
+-   ⬇️ Reduce the lead time for change
+-   💰 Reduce the cost of API integration testing
+-   💥 Prevent breaking changes
+-   🔎 Understand your system usage
+-   📃 Document your APIs for free
+-   🗄 Remove the need for complex data fixtures
+-   🤷‍♂️ Reduce the reliance on complex test environments
 
-## Known Issues
-1. When debugging a test locally (either consumer or provider) if you click the stop button in your test runner, it will abort the process abruptly and the ruby runtime will not get cleaned up. If you do this, simply kill the ruby process from your task/process manager. We recommend you play the test through to the end to avoid this issue. See https://github.com/pact-foundation/pact-net/issues/108 for more details.
+Watch our [series](https://www.youtube.com/playlist?list=PLwy9Bnco-IpfZ72VQ7hce8GicVZs7nm0i) on the problems with end-to-end integrated tests, and how contract testing can help.
 
-## Usage
-Below are some samples of usage.  
-For examples of Version 2 usage, please see the [Samples](https://github.com/pact-foundation/pact-net/blob/master/Samples/EventApi/Consumer.Tests/EventsApiConsumerTests.cs).  
+</td>
+</tr>
+</table>
 
-We have also written some `//NOTE:` comments inline in the code to help explain what certain calls do.
+![----------](https://raw.githubusercontent.com/pactumjs/pactum/master/assets/rainbow.png)
 
-**A few others things to note:**
-1. When using `Match.Regex` you must supply a valid Ruby regular expression, as we currently use the Ruby core engine.
+## Documentation
 
-### Installing
+This readme offers an basic introduction to the library. The full documentation for Pact Net and the rest of the framework is available at https://docs.pact.io/.
 
-Via Nuget  
+-   [Installation](#installation)
+-   [Consumer Testing](./docs/consumer.md)

```

Comment:
These links currently go to nowhere - did you forget to check them in?

Reply:
Missed this section during my edits. I'll remove for now and can treat this as a to-do list of later docs